### PR TITLE
Clean up acm governance policy creation tooltips

### PR DESCRIPTION
**Pull request by ChunxiAlexLuo**
https://github.com/open-cluster-management/backlog/issues/14873

Signed-off-by: Chunxi Luo chuluo@redhat.com

### CODE REVIEW

**Review by ycao56 — APPROVED** [review]
/lgtm
/hold until confirmed by @dockerymick

File: src-web/nls/platform.properties [hunk 1]
```diff
@@ -277,14 +277,14 @@ policy.violations.unknown = Unknown
...
-policy.create.remediation.tooltip = Represents how violations are handled as defined in the resource.
-policy.create.standards.tooltip = The name or names of security standards your policy should validate. You can only create a new label by adding it directly into the YAML file.
-policy.create.categories.tooltip = A security control category represent specific requirements for one or more standards. You can only create a new label by adding it directly into the YAML file.
-policy.create.controls.tooltip = The control contains the instructions for ensuring that a policy meets the security requirements for one or more standards. You can only create a new label by adding it directly into the YAML file.
-policy.create.disabled.tooltip = Select to disable the policy from being propagated to the managed cluster. If selected, the policy can be re-enabled from the dropdown menu in the policies table.
+policy.create.namespace.tooltip = Enter the hub cluster namespace where the policy is created and stored
```
Comment (dockerymick) [inline src-web/nls/platform.properties:280]: add a period (.) at the end of this statement

File: tests/jest/components/common/TemplateEditor/__snapshots__/TemplateEditor.test.js.snap [hunk 2]
```diff
@@ -140,7 +140,7 @@ exports[`TemplateEditor component renders as expected 1`] = `
                       id="pf-select-toggle-id-0-select-typeahead"
                       onChange={[Function]}
                       onClick={[Function]}
-                      placeholder="The namespace to create and store the policy on the hub cluster."
+                      placeholder="Enter the hub cluster namespace where the policy is created and stored"
```
Comment (dockerymick) [inline tests/jest/components/common/TemplateEditor/__snapshots__/TemplateEditor.test.js.snap:143]: period needed here

File: tests/jest/components/modules/__snapshots__/CreationView.test.js.snap [hunk 3]
```diff
@@ -785,7 +785,7 @@ exports[`CreationView module renders as expected with empty discovered data 1`]
                           autocomplete="off"
                           class="pf-c-form-control pf-c-select__toggle-typeahead"
                           id="pf-select-toggle-id-6-select-typeahead"
-                          placeholder="The namespace to create and store the policy on the hub cluster."
+                          placeholder="Enter the hub cluster namespace where the policy is created and stored"
```
Comment (dockerymick) [inline tests/jest/components/modules/__snapshots__/CreationView.test.js.snap:788]: period needed here

File: tests/jest/components/modules/__snapshots__/CreationView.test.js.snap [hunk 4]
```diff
@@ -1471,7 +1471,7 @@ exports[`CreationView module renders as expected without discovered data 1`] = `
                           autocomplete="off"
                           class="pf-c-form-control pf-c-select__toggle-typeahead"
                           id="pf-select-toggle-id-12-select-typeahead"
-                          placeholder="The namespace to create and store the policy on the hub cluster."
+                          placeholder="Enter the hub cluster namespace where the policy is created and stored"
```
Comment (dockerymick) [inline tests/jest/components/modules/__snapshots__/CreationView.test.js.snap:1474]: period needed here

**Review by dockerymick — CHANGES_REQUESTED** [review]
Thanks for making these changes, I left comments for the changes

**Review by dockerymick — APPROVED** [review]
Looks good to me! Thanks!

**Pull request merged**